doc/manual/source/protocols/store-path.md

```diff
@@ -82,7 +86,7 @@ where
 
   - if `type` = `"source:" ...`:
 
-    the hash of the [Nix Archive (NAR)] serialization of the [file system object](@docroot@/store/file-system-object.md) of the store object.
+    the [Nix Archive (NAR)] serialization of the [file system object](@docroot@/store/file-system-object.md) of the store object.
```

Comment:
Are you sure? I thought this was double-hashed.

Reply:
I have independent implementation that produces the same store paths as `"${./foo.txt}"` using single hashing.
I think that makes sense: if `type = "text"`, `inner-fingerprint` is a file content, if `type = "source'`, `inner-fingerprint` is a NAR of a path. And after that `inner-fingerprint` is hashed into `inner-digest`.

Reply:
OK fair enough :)